libs/template/labs/src/articles/storyblok/api/storyblok-api.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Observable } from 'rxjs';
+
+export const StoryblokToken = 'oryx.StoryblokToken';
+export const StoryblokApiService = 'oryx.StoryblokApiService';
+
+declare global {
+  interface Environment {
+    readonly ORYX_STORYBLOK_TOKEN?: string;
+  }
+}
+
+export interface StoryblokSearch {
+  slug?: string;
+  query?: string;
+}
+
+interface StoryblokContent {
+  content: {
+    component: string;
+    content: string;
+    description: string;
+    heading: string;
+    _uid: string;
+  };
+}
+
+export interface StoryblokEntryResponse {
+  story: StoryblokContent;
+}
+
+export interface StoryblokEntriesResponse {
+  stories: StoryblokContent[];
+}
+
+export interface StoryblokApiService {
+  getEntries(search: StoryblokSearch): Observable<StoryblokEntriesResponse>;
+  getEntry(search: StoryblokSearch): Observable<StoryblokEntryResponse>;
+}
+
+declare global {
+  interface InjectionTokensContractMap {
+    [StoryblokApiService]: StoryblokApiService;
```

Comment:
I actually think client was a better name, I just got my self confused initially. This stuff is not an api, so we should not name it an api.